packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -319,9 +319,23 @@ export interface IGetAvailableSlots {
       emoji?: string | undefined;
     }[]
   >;
+  teamMember?: string | undefined;
 }
 
 export async function getAvailableSlots({ input, ctx }: GetScheduleOptions): Promise<IGetAvailableSlots> {
+  const attendeeCalUsers = input.attendeeEmails
```

Comment:
We are already fetching original booking details in line 413. You can also get attendees from there instead of passing it via frontend.

Reply:
you're right, thanks for noticing it.